src/ensembl/src/global/globalHelper.ts

```diff
@@ -9,3 +9,17 @@ export const getBreakpoint = (width: number): BreakpointWidth => {
     return BreakpointWidth.SMALL;
   }
 };
+
+export const getQueryParamsMap = (queryStr: string) => {
+  const params = queryStr.replace('?', '').split('&'); // split all query param pairs
+
+  let paramMap: { [key: string]: string } = {};
+
+  params.forEach((param: string) => {
+    const [key, value] = param.split('='); // split param pair into key value
+
+    paramMap[key] = value; // add the param pair into the param map
+  });
+
+  return paramMap;
+};
```

Comment:
Not sure if for this PR, but do we possibly want to use a special library for parsing and generating query strings? Like [query-string](https://www.npmjs.com/package/query-string)?

Reply:
Yes, I think that'll be better.

Reply:
Will it just be a replacement for these few lines? I'm not sure if the rest of the functions like `stringify` will be useful for us.

Reply:
Why do you think stringify is not going to be useful for us? We can use it to:
- build query strings when updating the url in the browser window (e.g. genome browser url)
- build the query part of urls for api endpoints